.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "gui"]
+	path = gui
+	url = https://github.com/jeremypoulter/openevse_wifi_gui
```

Comment:
Would it be possible to update these repositories URL's to point to https://github.com/openevse/openevse_wifi_gui

Thanks